### ISSUE TITLE
Uncaught TypeError: object is not a function

### ISSUE THREAD

**Issue by ronag** [gaearon/react-hot-loader#54]
When running with react-hot-loader I'm getting:

```Uncaught TypeError: object is not a function```

At the last line of the following:

```javascript
/* WEBPACK VAR INJECTION */(function(module) {/* REACT HOT LOADER */ if (true) { (function () { var ReactHotAPI = __webpack_require__(9),     RootInstanceProvider = __webpack_require__(4);  if (typeof ReactHotAPI === "function" && typeof RootInstanceProvider === "object") { module.makeHot = module.hot.data ? module.hot.data.makeHot : ReactHotAPI(RootInstanceProvider.getRootInstances); } })(); }

module.exports = function(module) {
	if(!module.webpackPolyfill) {
		module.deprecate = function() {};
		module.paths = [];
		// module.parent = undefined by default
		module.children = [];
		module.webpackPolyfill = 1;
	}
	return module;
}


/* REACT HOT LOADER */ if (true) { (function () { module.hot.accept(function (err) { if (err) { console.error("Cannot not apply hot update to " + "module.js" + ": " + err.message); } }); module.hot.dispose(function (data) {   data.makeHot = module.makeHot; }); if (module.exports && module.makeHot) { var freshModuleExports = module.exports; module.exports = module.makeHot(module.exports, "__MODULE_EXPORTS"); for (var key in module.exports) { if (freshModuleExports[key] && freshModuleExports.hasOwnProperty(key)) { module.exports[key] = module.makeHot(freshModuleExports[key], "__MODULE_EXPORTS_" + key); } } } })(); }
/* WEBPACK VAR INJECTION */}.call(exports, __webpack_require__(6)(module)))
```

**Comment by gaearon** [comment]
Are you using the latest version? Try 1.0.6.
If it doesn't help, change your config to have `exclude: /node_modules`, e.g.

```js
{ test: /\.jsx?$/, loaders: ['react-hot', 'jsx?harmony'] , exclude: /node_modules/ },
```

**Comment by gaearon** [comment]
Related webpack/webpack#632.

**Comment by ronag** [comment]
exclude helped.

**Comment by gaearon** [comment]
Are you on Windows?

**Comment by ronag** [comment]
Correct.

**Comment by gaearon** [comment]
OK, I know what this is. Thanks.

I have an [additional check](https://github.com/gaearon/react-hot-loader/blob/master/index.js#L15) to guard against this but it uses `/` as separator.

I'm not sure whether to fix the check, or to remove it altogether and tell everyone to use `exclude`..

**Comment by gaearon** [comment]
Fixed in 1.0.7. Thanks for reporting!
